src/exp/istio-watcher/config/config.go

```diff
@@ -0,0 +1,33 @@
+package config
+
+import (
+	"github.com/spf13/viper"
+	"strings"
+	"time"
+)
+
+const (
+	EnvPrefix               = "OTTERIZE"
```

Comment:
isn't this a duplicate? the keys here seem to belong to the network mapper, at least some of them